### PR TITLE
Adds Web Servers samples

### LINKED ISSUE #322: Add Web Servers Samples
Now that [Web Servers RFC 0005](https://github.com/paketo-buildpacks/rfcs/blob/3e2a00010fd077c10186671ce72ff5e227a2adb0/text/web-servers/0005-promote-web-servers-buildpack.md) has been accepted, this repo should contain samples using the Web Servers buildpack.

- [x] A new `web-servers` top level directory should be created.
- [x] The `httpd` samples should be moved into this directory.
- [x] A copy of `nginx` samples should be added to the directory. (Since [httpd does not run on the base builder](https://github.com/paketo-buildpacks/httpd/issues/322), a copy of the`nginx` samples should also be kept in their own top level directory, so they can [continue to be tested against the base builder](https://github.com/paketo-buildpacks/samples/blob/2585fe48279451e638bc3f826a6d43b58e8e0fc8/.github/workflows/test-pull-request-nginx.yml#L25).)
- [x] The [react sample](https://github.com/paketo-buildpacks/samples/blob/2585fe48279451e638bc3f826a6d43b58e8e0fc8/nodejs/react-httpd-nginx/README.md) should be moved to the web servers directory.
- [x] Be sure to follow the instructions in the Adding New Samples section of the [readme](https://github.com/paketo-buildpacks/samples#adding-new-samples)

### PR DESCRIPTION
**Pull request by ForestEckhardt**
Resolves #322

### CODE REVIEW

**Comment by ForestEckhardt** [comment]
Needed for RFC 0047 https://github.com/paketo-buildpacks/rfcs/issues/193

**Comment by ForestEckhardt** [comment]
Test is failing waiting for addition of Web Server buildpack to the full builder

File: web-servers/no-config-file-sample/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Zero Configuration Static File App
+This directory hass all of the components to for a sample of a zero
```
Comment (TisVictress) [inline web-servers/no-config-file-sample/README.md:2]: On line 2, `has` is misspelled and you don't need `to` after components.

**Review by TisVictress — CHANGES_REQUESTED** [review]
This LGTM :-) Just some misspellings but otherwise I approve.

**Review by TisVictress — APPROVED** [review]
LOL you made the changes already.

**Review by thitch97 — APPROVED** [review]

**Pull request merged**